Rakefile

```diff
@@ -4,21 +4,27 @@ require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
 
-RSpec::Core::RakeTask.new(:specs)
-
-task default: :specs
+task default: :all
 
-desc 'Runs all tasks: :specs, :rubocop and :spec_docs'
-task :spec do
+desc 'Runs all tasks: :specs, :rubocop and :danger_lint'
+task :all do
   Rake::Task['specs'].invoke
-  Rake::Task['rubocop'].invoke
-  Rake::Task['spec_docs'].invoke
+  Rake::Task['lint'].invoke
 end
```

Comment:
Instead of calling `Rake::Task[…].invoke` on each one, you should just declare those tasks as dependencies to the parent task, which would make this all more compact (no need for `do…end`). iirc the syntax should look something like:

```
task :all => %i[specs lint]
task :lint => %i[rubocop danger_lint]
```

Reply:
Nice one! Committed here 69f7d299a3432c1a557e32c51d32711853605d01